AEM/mlperf/README.md

```diff
@@ -1,4 +1,213 @@
-# Get Started with Intel MLPerf v3.1 Submission with Intel Optimized Docker Images
+# Get Started with Intel MLPerf v4.0 Submission with Intel Optimized Docker Images
+
+MLPerf is a benchmark for measuring the performance of machine learning
+systems. It provides a set of performance metrics for a variety of machine
+learning tasks, including image classification, object detection, machine
+translation, and others. The benchmark is representative of real-world
+workloads and as a fair and useful way to compare the performance of different
+machine learning systems.
+
+
+In this document, we'll show how to run Intel MLPerf v4.0 submission with Intel
+optimized Docker images.
+
+## HW configuration:
+
+| System Info     | Configuration detail                 |
+| --------------- | ------------------------------------ |
+| CPU             | EMR                                  |
+| OS              | CentOS  Stream 8                     |
+| Kernel          | 6.6.8-1.el8.elrepo.x86_64            | 
+| Memory          | 1024GB (16x64GB 5600MT/s [5600MT/s]) |
+| Disk            | 1TB NVMe                             |
+
+## BIOS settings:
+| BIOS setting    | Recommended value                    |
+| --------------- | ------------------------------------ |
+|Hyperthreading|Enabled
+|Turbo Boost|Enabled
+|Core Prefetchers|Hardware,Adjacent Cache,DCU Streamer,DCU IP
+|LLC Prefetch|Disable
+|CPU Power and Perf Policy|Performance
+|NUMA-based Cluster|SNC2
+|Hardware P State|Native (based on OS guidance)
+|Energy Perf Bias|OS Controls EPB
+|Energy Efficient Turbo|Disabled
+
```

Comment:
```suggestion
## Check System Health Using Intel® System Health Inspector:
Intel® System Health Inspector (aka svr-info) is a Linux OS utility for assessing the state and health of Intel Xeon computers. It is suggested to use svr-info first to check any system configuration issue before running any benchmark. Follow [the Quick Start Guide](https://github.com/intel/svr-info#quick-start) for downloading and installation. The following are several key factors effecting the model performance.

<details>
<summary> CPU </summary>
Couple CPU features impact MLPerf performance via related BIOS knobs, so please double check the CPU features with your BIOS knobs.
Some important CPU features are Hyperthreading, number of NUMA nodes, Prefetchers and Intel Turbo Boost.
<br><img src="BIOS_examples/CPU_setting.png" width="300" height="600"><br>
</details>

<details>
<summary> Memory </summary>
One important system configuration is balanced DIMM population, which is suggested to set as balanced to get optimized performance. <br> 
Populate as many channels per socket as possible prior to adding additional DIMMs to the channel.   
It might impact the memory bandwidth if two dimm share one channel. <br>   
Please also refer to Chapter 4 in <a href="https://cdrdv2.intel.com/v1/dl/getContent/733546?explicitVersion=true">Eagle Stream Platform Performance & Power Optimization Guide</a> for more details.  <br> 
     
From the results of svr-info, an example of unbalanced DIMM population is shown as follows,
<br><img src="BIOS_examples/Unbalanced_DIMM.png" width="300" height="600"><br>

An exmaple of Balanced DIMM population is shown as follows,     
<br><img src="BIOS_examples/Balanced_DIMM.png" width="300" height="600"><br>

You should also see good numbers for memory NUMA bandwidth if you also benchmark memory via svr-info. <br>
Here are some reference numbers from a 2S SPR system.
<br><img src="BIOS_examples/mem_bandwidth.png" width="200" height="150"><br>     
     
</details>

<details>
<summary> Power  </summary>
We recommend the intel_pstate Frequency Driver. <br>
For best performance, set the Frequency Governor and Power and Perf Policy to performance. <br>
Here are related recommended power settings from svr-info. 
<br><img src="BIOS_examples/power_setting.png" width="400" height="300"><br>
</details>

```